language-support/ts/codegen/tests/src/DA/Test/Daml2js.hs

```diff
@@ -14,31 +14,21 @@ import qualified DA.Daml.LF.Ast.Version as LF
 import DA.Test.Daml2jsUtils
 import Data.List.Extra
 import qualified Data.Text.Extended as T
-import qualified Data.ByteString.Lazy as BSL
```

Comment:
@stefanobaghino-da can you please take a look if that looks reasonable to you? Factored it into a separate commit for easier review 4ec55da95bcf47e844d3b709be090d6d78e49e8a. I just dropped everything that depends on LF < 1.8.

Reply:
I tried to go through it but my Haskell and knowledge of `daml2js` is insufficient to give you a meaningful answer. Who owned this previously? All I can say is that I would trust whomever actively maintained this previously, as to the best of my understanding there hasn't been any handover for this. If the application runtime need to own it, we'll probably need to rewrite it at some point as I don't think we should spread ourselves too thin into maintaining more and more stuff far away from the core skills of the team.

Reply:
Martin & Robin wrote it originally. since then I did some of the work and Gary made some changes to support genmaps.